src/components/Proposals/ProposalDetails/StatusInfo.js

```diff
@@ -24,6 +26,37 @@ function StatusInfo({ vote }) {
       </Info>
     )
   }
+
+  if (disputableStatus === VOTE_STATUS_SETTLED) {
+    return (
+      <Info
+        mode="warning"
+        css={`
+          margin-top: ${2 * GU}px;
+        `}
+      >
+        This vote has been canceled as the result of the originating action
+        being challenged and the settlement offer being accepted.
+      </Info>
+    )
+  }
```

Comment:
We should create a mapping for these descriptions based on the status, e.g:
```javascript
const infoDescription = {
    [VOTE_STATUS_SETTLED]: 'This vote has been canceled as the result of the originating action
        being challenged and the settlement offer being accepted.',
    [VOTE_STATUS_DISPUTED]: '....',
}
```

And then our template can be as simple as:

```javascript
<Info
     mode="warning"
     css={`
       margin-top: ${2 * GU}px;
     `}
   >
     {infoDescription[disputableStatus]}
   </Info>
```